mne/beamformer/_compute_beamformer.py

```diff
@@ -173,12 +194,15 @@ def _normalized_weights(Wk, Gk, Cm_inv_sq, reduce_rank, nn, sk):
     Wk_max = np.matmul(max_power_ori[:, np.newaxis], Wk)[:, 0]
```

Comment:
We still call things `Wk`, `Gk`, etc here, but it seems things have been changed and we are actually not iterating over sources anymore @larsoner ?

Reply:
Indeed these could just be called W, G, etc. If you want to rename you should wait for #7168 to be merged, rebase, and rename. Though it might be easiest if the renaming was the only thing done in the PR so that it's clear what meaningfully rather than aesthetically changed.

Reply:
Yes, I agree this should be an extra PR - let's do it after this one and the `reduce_rank` one together with removing the context manager. I think it would be clearer, especially when only reading the code.